crates/wdk-build/src/lib.rs

```diff
@@ -243,6 +243,10 @@ rustflags = [\"-C\", \"target-feature=+crt-static\"]
     /// [`metadata::Wdk`]
     #[error(transparent)]
     SerdeError(#[from] metadata::Error),
+
+    /// Error returned when the UCX header file is not found
+    #[error("failed to find ucx header file")]
+    UCXHeaderNotFound(#[source] std::io::Error),
```

Comment:
nit: Should this be more generic instead of just for UCX? extra metadata can be in the error (like api subset or the path that doesn't exist